ptbcontrib/username_to_chat_api/README.md

```diff
@@ -4,28 +4,62 @@ This provides a wrapper for the [usernameToChatAPI](https://github.com/Poolitzer
 
 The API uses an userbot in the background to obtain the information for the `Chat` object. This is not possible with the plain HTTP Bot API (and that is the reason why this API exists).
 ```python
+import asyncio
+
 from ptbcontrib.username_to_chat_api import UsernameToChatAPI
 from telegram import Bot, error
 
 import time
 
 bot = Bot("BOT_TOKEN")
-# or you could get it from updater.bot
 wrapper = UsernameToChatAPI("https://localhost:1234/", "RationalGymsGripOverseas", bot)
-# this could be saved to bot_data, but should only be initiated once
 try:
-    chat = wrapper.resolve("@poolitzer")
+    chat = asyncio.run(wrapper.resolve("@poolitzer"))
 except error.RetryAfter as e:
     time.sleep(e.retry_after)
     # both variants work
-    chat = wrapper.resolve("poolitzer")
+    chat = asyncio.run(wrapper.resolve("poolitzer"))
+```
 
+But there is more: This implements itself even nicer into a PTB application with a custom context:
+```python
+import logging
 
-```
+from telegram import Update, Chat
+from telegram.ext import ApplicationBuilder, CallbackContext, CommandHandler, ContextTypes, Application
+from ptbcontrib.username_to_chat_api import UsernameToChatAPI
+
+logging.basicConfig(
+    format='%(asctime)s - %(name)s - %(levelname)s - %(message)s',
+    level=logging.INFO
+)
+
+
+class CustomContext(CallbackContext):
+    def __init__(self, application: Application):
+        super().__init__(application=application)
+        self._wrapper: UsernameToChatAPI = UsernameToChatAPI("https://localhost:1234/", "RationalGymsGripOverseas", application.bot)
```

Comment:
do you really want to initialize a new instance every time a `context` object is build? o.0

Reply:
No I dont you are right. Is there a way to store the instance somewhere, in the application maybe?  otherwise I guess a custom ContextType isnt a good way to go

Reply:
class variable? `bot_data`?